src/PIL/Image.py

```diff
@@ -1387,7 +1393,7 @@ def getdata(self, band=None):
             return self.im.getband(band)
         return self.im  # could be abused
 
-    def getextrema(self):
+    def getextrema(self) -> tuple[float, float] | tuple[tuple[float, float], ...]:
```

Comment:
```suggestion
    def getextrema(self) -> tuple[float, float] | tuple[tuple[int, int], ...]:
```
We don't have multi-band images with float values.